<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectUserInterface</class>
 <widget class="QWidget" name="SelectUserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userSignLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);
color: rgb(0, 85, 0);
font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>用户列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>23456</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>34567</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>45678</string>
   </property>
  </widget>
  <widget class="QPushButton" name="common_chat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>发起群聊</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
